apps/client/src/routes/dashboards/dashboards-index/styles.css

```diff
@@ -0,0 +1,7 @@
+@import '../../../styles/variables.css';
+
+.btn-create-dashboard {
```

Comment:
name this as `btn-custom-primary` or something along those lines

Reply:
done